variables.tf

```diff
@@ -0,0 +1,15 @@
+variable "namespace" {}
+
+variable "stage" {}
+
+variable "name" {}
+
+variable "delimiter" {}
+
+variable "attributes" {}
```

Comment:
Use variable definitions of [tf_label](https://github.com/cloudposse/tf_label/blob/master/variables.tf)